e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -72,6 +72,29 @@ describe('I18456 Built-in slash commands: common', () => {
             cy.get(`#postMessageText_${postId}`).should('have.text', 'test ¯\\_(ツ)_/¯');
         });
     });
+
+    it('MM-T710 /mute error message', () => {
+        loginAndVisitDefaultChannel(user1, testChannelUrl);
+
+        const invalidChannel = 'oppagangnamstyle';
+
+        // # Type /mute with random characters
+        cy.postMessage(`/mute ${invalidChannel}`);
+        cy.wait(TIMEOUTS.THREE_SEC);
+
+        cy.getLastPostId().then((postId) => {
+            // * Could not find the channel lalodkjngjrngorejng. Please use the channel handle to identify channels.
+            cy.get(`#postMessageText_${postId}`).should('have.text', `Could not find the channel ${invalidChannel}. Please use the channel handle to identify channels.`);
+        });
+
+        // * Channel handle links to: https://docs.mattermost.com/help/getting-started/organizing-conversations.html#naming-a-channel
+        cy.contains('a', 'channel handle').then((link) => {
```

Comment:
The link from the channel handle link is `https://about.mattermost.com/default-channel-handle-documentation`. This link redirects to `https://docs.mattermost.com`.

Reply:
Yes, that's fine and correct. 

By the way, this check should be scoped within the last post. Can be chained as,
```javascript
cy.getLastPostId().then((postId) => {
    cy.get(`#postMessageText_${postId}`).
        should('have.text', `Could not find the channel ${invalidChannel}. Please use the channel handle to identify channels.`).
        contains('a', 'channel handle').then((link) => {
            const href = link.prop('href');
            cy.request(href).its('allRequestResponses').then((response) => {
                cy.wrap(response[1]['Request URL']).should('equal', 'https://docs.mattermost.com/help/getting-started/organizing-conversations.html#naming-a-channel');
            });
        });
});
```